web/src/app/globals.css

```diff
@@ -1,5 +1,6 @@
-@import 'tailwindcss/theme.css';
-@import 'tailwindcss/utilities.css';
+/* you might want to consider adding this preflight line here. it enables tailwind's preflight. we can't do it by default in our other apps because they are relying on their own css reset but if you can it will make your app look more like "standard" tailwind */
+/* @import "tailwindcss/preflight.css" layer(base); */
```

Comment:
if you can use tailwind's preflight, I would. we can't in our other apps but you should if you can